blather.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.rdoc_options = %w{--charset=UTF-8}
   s.extra_rdoc_files = %w{LICENSE README.md}
 
-  s.add_dependency "eventmachine", ["~> 1.0.9"] # https://github.com/eventmachine/eventmachine/issues/670
+  s.add_dependency 'eventmachine', ['~> 1.2', '>= 1.2.6']
```

Comment:
Can we be consistent about quotes?

Reply:
I'd like to be but it's already inconsistent elsewhere and RuboCop would obviously like single quotes if we were to apply it.